tests/Makefile

```diff
@@ -13,7 +13,7 @@ endif
 
 # NOTE: This assumes that a working `ghc` is on $PATH; this may not necessarily be the same GHC used by `cabal` for building `alex`.
 HC=ghc
-HC_OPTS=-Wall -fno-warn-missing-signatures -fno-warn-unused-imports -fno-warn-tabs -Werror
+HC_OPTS=-Wall -fwarn-incomplete-patterns -fwarn-incomplete-uni-patterns -fno-warn-missing-signatures -fno-warn-unused-imports -fno-warn-tabs -Werror
```

Comment:
These warnings are included in `-Wall`, so you can remove them from the argument list

Reply:
I removed the `incomplete-patterns` warning. The documentation for `-Wall` includes `incomplete-uni-patterns` as one of the warnings it doesn't enable, so I left it.